packages/opentelemetry-tracing/src/BasicTracerProvider.ts

```diff
@@ -78,7 +83,9 @@ export class BasicTracerProvider implements api.TracerProvider {
   register(config: SDKRegistrationConfig = {}) {
     api.trace.setGlobalTracerProvider(this);
     if (config.propagator === undefined) {
-      config.propagator = new HttpTraceContext();
+      config.propagator = new CompositePropagator({
```

Comment:
Currently, `HttpTraceContext` is highlighted as a default propagator in the documentation (https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-core#built-in-propagators). Could you please update the same?

Reply:
Hi, I added a new commit to change the documentation. Thanks for pointing this out.